### DISCUSSION TITLE
Slider component is REALLY difficult to use

### DISCUSSION THREAD

**Discussion by devjones** [advplyr/audiobookshelf-app#366] [Ideas]
The biggest issue for me now is the slider component.  To move it, my thumb has to remain directly over the component at the current time stamp.  This makes it really tricky to navigate a few mins back in a long book and get precision.

Other apps, like Prologue or Apple Music, you can continue to control the slider as you move your thumb directly off the slider.

I do this rewind action often in a single audiobook

**Comment by advplyr** [comment]
The slider should still work with your thumb off the slider. I recently changed the UI adding the circular cursor so now it looks weird though, I think I will update this to move the white line and cursor instead of how it works now.

![hold_and_drag_mobile](https://user-images.githubusercontent.com/67830747/188475260-444a838a-bc75-454b-a852-709d531e0807.gif)

**Reply by advplyr** [reply]
I also wonder now if we should force the user to click the cursor. Some users are reporting accidently jumping back by pressing the track and this could be reduced if we force grabbing the cursor.
